### ISSUE TITLE
There is sensible content on the privacy policy page

### ISSUE THREAD

**Issue by r-ferrier** [geeksforsocialchange/gfsc-v3#250]

**Comment by aaaaargZombies** [comment]
## :partying_face:  thirdpartys :partying_face:  

### mailerlite

- news letter sign up, their [privacy policy](https://www.mailerlite.com/legal/privacy-policy)

### spotify for podcasts

- podcast iframe used to play the podcast on our site
- adds 4 cookies on page load
- [privacy policy](https://podcasters.spotify.com/pod/privacy)

### youtube

- we embed youtube iframes in blog posts and projest
- there are many cookies
![2023-06-27-12:43:57-screenshot](https://github.com/geeksforsocialchange/gfsc-v3/assets/6824891/35506591-5ded-4035-822a-fd2516a94799)
- harder to track down specific privacy policy

### adobe / typekit

- we pull in balboa and frieght-text-pro here is their [privacy policy](https://www.adobe.com/privacy/policies/adobe-fonts.html)

**Comment by kimadactyl** [comment]
i vaguely remember there being a cookie-less youtube option but im damned if i can remember what it was

**Comment by r-ferrier** [comment]
Have started this and have written a draft policy here, based on the one on the trans dimension: https://github.com/geeksforsocialchange/gfsc-v3/tree/feat/250-add-privacy-policy

HOWEVER have run into an unfortunate barrier: spotify embeds do not conform to this policy. Users must consent to the use of analytics cookies before we can show them. We also currently say that we collect anonymous data using plausible analytics (nbot sure if this is true) but if it is true, even if it is anonymous, AFAIK we still need to ask for visitors consent to collect this. I will raise a new issue about these things.

**Comment by r-ferrier** [comment]
New issue here: https://github.com/geeksforsocialchange/gfsc-v3/issues/287

**Comment by r-ferrier** [comment]
@aaaaargZombies @ivan-kocienski-gfsc @katjam any thoughts on the above?

**Comment by ivan-kocienski-gfsc** [comment]
Seems okay to me (but I am not a legal person so...)

Perhaps change the email address from Kim's ?

**Comment by r-ferrier** [comment]
Have made a [proposal for dealing with cookies on the website](https://www.notion.so/gfsc/Remove-youtube-embeds-from-our-website-and-replace-spotify-embeds-with-our-own-web-component-aa8e52baf90e4ef7b49dce557f16edd6?pvs=4) and this will remain blocked until this is agreed on and https://github.com/geeksforsocialchange/gfsc-v3/issues/287 is sorted out.

**Comment by r-ferrier** [comment]
### This page explains how we collect information via gfsc.studio and what we do with it.

### What data do we collect?

GFSC collects anonymous analytics data.

We sometimes use third party services for example [Plausable Analytics](https://plausible.io/privacy), to gather statistical data about the use of gfsc.studio. That data isn't linked to you in any way.

- We use [Spotify](https://www.youtube.com/) to embed podcasts on our website. Spotify use cookies to gather analytics. [Spotify privacy policy](https://www.spotify.com/uk/legal/privacy-policy/)

- We embed youtube iframes in blog posts and projects using youtube's 'privacy enhanced mode'. These load functional cookies required to run videos but do not gather data that can be linked to any actions you take on our site. [Youtube API ToS](https://developers.google.com/youtube/terms/api-services-terms-of-service)


- We use adobe typekit to serve our fonts. [Adobe privacy policy](https://www.adobe.com/privacy/policy.html)

#### Information to help us get in touch with you
In connection with services offered through this site we may ask you to submit information such as your name and e-mail address. You are under no obligation to provide such information, however should you not provide such information we may not be able to offer you the following services:

- Sign-up to newsletter. We use [mailerlite](https://www.mailerlite.com) to manage our newsletter. [mailerlite privacy policy](https://www.mailerlite.com/legal/privacy-policy)

### How is the information used?

#### GFSC may use the information you provide to
- Improve the content of our web page
- Send you any information you have requested, including our newsletter
- We may occasionally send you information about news and events we feel may be of interest to you. This will only come from us and only if you have requested this information.
- You can also tell us if you no longer wish to receive information from GFSC.

#### GFSC uses anonymous analytics data to
- Monitor the number of visitors to our website
- Monitor how users use our website, i.e. which pages they have accessed
- To help improve our website

#### Who is the information shared with?
GFSC may analyse and disclose statistics about the number of visitors to our website in order to describe our services to prospective partners, other reputable third parties and for other lawful purposes. These statistics do not include and personal or identifiable information about visitors.

GFSC may disclose personal information if required to do so by law or if it believes in good faith that such action is required by law.

GFSC contains links to other sites. GFSC is not responsible for the privacy policies or the content of such sites.

### Your acceptance of these terms

By using this site, you consent to the collection and use of this information by GFSC and to the GFSC privacy policy.

If you have any questions or concerns about your privacy when using our website, please contact us: [kim@gfsc.studio](mailto:kim@gfsc.studio)

**Comment by katjam** [comment]
That draft looks good and much better than ignoring it. I agree with ivan though, we should probably move towards using a privacy@gfsc.studio or info@gfsc.studio address so that Kim is not a single point of responsibility.
